### PR TITLE
Small PCM updates after scfitertopy

### LINKED ISSUE #999: PCM-ROHF not working
PCM-ROHF is currently broken. A fix is in the pipeline with #953, but is sufficiently invasive that it **will not make it** to the upcoming 1.2 release.

### PR DESCRIPTION
**Pull request by robertodr**
## Description
Some PCM related updates/fixes that were enabled after the move of SCF to Python.

## Todos
Notable points (developer or user-interest) that this PR has or will accomplish.
- [x] Tests ROHF with PCM. Close #999 
~- [ ] Do gradients with PCM semi-numerically [à la ECP](https://github.com/psi4/psi4/blob/master/psi4/driver/procrouting/proc.py#L2095-L2132), see https://github.com/psi4/psi4/issues/402#issuecomment-386149151~ **Probably for 1.4**

## Checklist
- [x] Tests added for any new features
- [x] [All or relevant fraction of full tests run](http://psicode.org/psi4manual/master/build_planning.html#how-to-run-a-subset-of-tests)

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

**Comment by loriab** [comment]
Will the existing testing close #999? Do you want to get the existing todo item in for 1.3, and defer the findif to later?

File: tests/pcmsolver/scf/input.dat [hunk 1]
```diff
@@ -63,6 +63,14 @@ energy_scf, wfn = energy('scf', return_wfn=True)
 compare_values(totalenergy, energy_scf, 10, "Total energy (PCM, separate algorithm)") #TEST
 compare_values(polenergy, wfn.get_variable("PCM POLARIZATION ENERGY"), 6, "Polarization energy (PCM, separate algorithm)")  #TEST
 
+# Now force use of ROHF on NH3 to check sanity of the algorithm with PCM
+set pcm_scf_type total
+set reference rohf
+print('PK-ROHF-PCM, total algorithm')
```
Comment (andysim) [inline tests/pcmsolver/scf/input.dat]: maybe `print_out` makes more sense here

File: tests/pcmsolver/scf/input.dat [hunk 2]
```diff
@@ -63,6 +63,14 @@ energy_scf, wfn = energy('scf', return_wfn=True)
...
+energy_scf, wfn = energy('scf', return_wfn=True)
+compare_values(totalenergy, energy_scf, 10, "Total energy (PCM, separate algorithm)") #TEST
+compare_values(polenergy, wfn.get_variable("PCM POLARIZATION ENERGY"), 6, "Polarization energy (PCM, separate algorithm)")  #TEST
```
Comment (loriab) [inline tests/pcmsolver/scf/input.dat]: `get_variable` --> `variable` to fend off the warning. here and below.

Comment (robertodr) [reply tests/pcmsolver/scf/input.dat]: Sure thing :smile_cat:

**Pull request merged**